content/markdown/concepts/emphasis/emphasis.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Emphasis'
-Description: 'Emphasis can be added by making text bold or italic. To bold text, add two asterisks  or underscores __ before and after a word or phrase. To bold the middle of a word for emphasis, add two asterisks without spaces around the letters.'
+Description: 'Applies importance to text by making it bold, italic, and/or strikethrough.'
```

Comment:
So this is a regular entry, not a Term entry. So the Description should still start with the noun and not the verb.

```suggestion
Description: 'Emphasis applies importance to text in Markdown by making it bold, italic, and/or strikethrough.'
```

Reply:
that's all from my end!